azext_edge/tests/edge/support/conftest.py

```diff
@@ -15,12 +15,12 @@ def add_pod_to_mocked_pods(
     mocked_client,
     expected_pod_map,
     mock_names: List[str] = None,
-    mock_init_containers: bool = False
+    mock_init_container: bool = False
 ):
     from kubernetes.client.models import V1PodList, V1Pod, V1PodSpec, V1ObjectMeta, V1Container
 
     current_pods = mocked_client.CoreV1Api().list_pod_for_all_namespaces.return_value
-    pod_list = current_pods.items()
+    pod_list = current_pods.items
```

Comment:
is current_pods supposed to be a dict? make sure that items does not need the ()

Reply:
yep. items() actually report errors when called(the function wasn't triggered before so didn't found the error), V1pod only has items property